test/pom.xml

```diff
@@ -280,6 +280,14 @@ THE SOFTWARE.
           </rules>
         </configuration>
       </plugin>
+      <plugin> <!-- https://stackoverflow.com/a/19503679/12916 ysoserial deliberately uses internal APIs; suppress warnings about them -->
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-compiler-plugin</artifactId>
```

Comment:
We may need to inherit configuration from parent POM at some point, good for now